x-pack/plugins/secops/server/lib/overview/query.dsl.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { createQueryFilterClauses } from '../../utils/build_query';
+import { RequestBasicOptions } from '../framework';
+
+export const buildOverviewNetworkQuery = ({
+  filterQuery,
+  timerange: { from, to },
+  sourceConfiguration: {
+    fields: { timestamp },
+    auditbeatAlias,
+    logAlias,
+    packetbeatAlias,
+  },
+}: RequestBasicOptions) => {
+  const filter = [
+    ...createQueryFilterClauses(filterQuery),
+    {
+      range: {
+        [timestamp]: {
+          gte: from,
+          lte: to,
+        },
+      },
+    },
+  ];
+
+  const dslQuery = {
+    allowNoIndices: true,
+    index: [auditbeatAlias, logAlias, packetbeatAlias],
+    ignoreUnavailable: true,
+    body: {
+      aggregations: {
+        unique_flow_count: {
+          filter: {
+            term: { type: 'flow' },
+          },
+        },
+        unique_dns_count: {
+          filter: {
+            term: { type: 'dns' },
+          },
+        },
+        unique_suricata_count: {
+          filter: {
+            term: { 'service.type': 'suricata' },
+          },
+        },
+        unique_zeek_count: {
+          filter: {
+            term: { 'service.type': 'zeek' },
+          },
+        },
+        unique_socket_count: {
+          filter: {
+            term: { 'event.dataset': 'socket' },
+          },
+        },
+      },
+      query: {
+        bool: {
+          filter,
+        },
+      },
+      size: 0,
+      track_total_hits: true,
+    },
+  };
+
+  return dslQuery;
+};
+
+export const buildOverviewHostQuery = ({
+  filterQuery,
+  timerange: { from, to },
+  sourceConfiguration: {
+    fields: { timestamp },
+    auditbeatAlias,
+    logAlias,
+    packetbeatAlias,
+  },
+}: RequestBasicOptions) => {
+  const filter = [
+    ...createQueryFilterClauses(filterQuery),
+    {
+      range: {
+        [timestamp]: {
+          gte: from,
+          lte: to,
+        },
+      },
+    },
+  ];
+
+  const dslQuery = {
+    allowNoIndices: true,
+    index: [auditbeatAlias, logAlias, packetbeatAlias],
```

Comment:
@andrew-goldstein @FrankHassanabad - you had me add all 3 aliases. Can you please explain why? @XavierM is helping me with the es-archiver stuff and thinks I should not be querying anything except auditbeat

Reply:
Since this doesn't query anything from those three at the moment it is fine to cut this down to just one alias which is `auditbeatAlias`

Later since these will be configurable through https://github.com/elastic/ingest-dev/issues/251, will change all of these again.